etl/steps/data/garden/wb/2023-12-29/gender_statistics.meta.yml

```diff
@@ -13,30 +13,68 @@ tables:
     # Learn more about the available fields:
     # http://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/
     variables:
-      total_maternity_leave:
-        title: Total days of leave available for the mother
+      total_maternity_leave_to:
+        title: Total days of leave available to the mother
         unit: days
         short_unit: ''
-        description_processing: The total days of leave available for the mother indicator was created by OWID by combining maternity leave and parental leave mother quota indicators. For more details on each indicator see the section on **How producer describes this data**.
+        description_processing: This indicator was created by OWID by combining maternity leave and parental leave mother quota indicators. For more details on each indicator see the section on **How producer describes this data**.
```

Comment:
I think it's better to use Our World in Data instead of OWID. Same for the other indicators.